src/data/web3.ts

```diff
@@ -10,14 +10,14 @@ import icon from '@/assets/images/near_social_icon.svg';
 
 const web3onboardKey = 'web3-onboard:connectedWallets';
 
-const wcV1InitOptions = {
-  qrcodeModalOptions: {
-    mobileLinks: ['metamask', 'argent', 'trust'],
-  },
-  connectFirstChainId: true,
+const wcV2InitOptions: any = {
+  version: 2,
+  // TODO: add proper `projectId`
+  projectId: '',
```

Comment:
@marcinbodnar  we can use `72b7b3359ab477e339a070f615806aa6`

Reply:
Done.